src/main.js

```diff
@@ -227,12 +228,30 @@ app.on('ready', function() {
 
     trayIcon.setToolTip(app.getName());
     trayIcon.on('click', function() {
+      if (process.platform === 'win32') {
+        if (config.minimizeToTray) {
+          if (mainWindow.isHidden) {
```

Comment:
> However, the result of a single click with the left mouse button will vary with the function of the icon. It should display what the user would expect to see in the form best suited to that content—a popup window, a dialog box or the program window itself. For instance, it could show status text for a status icon, or a slider for the volume control.

On [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ee330740(v=vs.85).aspx), it seems that a single click is expected as "show" function.